invenio_communities/views/ui.py

```diff
@@ -149,11 +149,22 @@ def create_ui_blueprint(app):
         strict_slashes=False,
     )
 
+    blueprint.add_url_rule(
+        routes["deprecated_search"],
+        view_func=communities_search,
+        strict_slashes=False,
+    )
+
     blueprint.add_url_rule(
         routes["new"],
         view_func=communities_new,
     )
 
+    blueprint.add_url_rule(
+        routes["deprecated_new"],
+        view_func=communities_new,
```

Comment:
How do we communicate the deprecation? I assume it will be part of the release notes but does it make sense to log a deprecation warning in the view method so instance administrator see it if there are still occurrences in their code? ping @slint @ntarocco

Reply:
It should go in the release notes. Agree, logging a warning is probably the best for visibility, though there should also be the possibility to do something about it (e.g. suggest that if the instance wants to keep the endpoint they should handle a redirect on their side).